python/cucim/src/cucim/skimage/data/_binary_blobs.py

```diff
@@ -45,19 +49,16 @@ def binary_blobs(
     --------
     >>> from cucim.skimage import data
     >>> # tiny size (5, 5)
-    >>> blobs = data.binary_blobs(length=5, blob_size_fraction=0.2, seed=1)
+    >>> blobs = data.binary_blobs(length=5, blob_size_fraction=0.2)
```

Comment:
Should we use `rng` here?

```suggestion
    >>> blobs = data.binary_blobs(length=5, blob_size_fraction=0.2, rng=1)
```

Reply:
it isn't necessary as we aren't doing any test on specific output values